modules/Microsoft.RecoveryServices/vaults/replicationAlertSettings/deploy.bicep

```
@@ -0,0 +1,59 @@
+@description('Required. The name of the parent Azure Recovery Service Vault.')
```

For consistency with other child modules I'd suggest
```suggestion
@description('Conditional. The name of the parent Azure Recovery Service Vault. Required if the template is used in a standalone deployment.')
```